torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -36,7 +36,7 @@ struct RpcBackendOptions {
 };
 
 // A globally unique ID to identify an RpcAgent
-struct TORCH_API WorkerInfo {
+struct TORCH_API WorkerInfo : torch::CustomClassHolder {
   WorkerInfo(std::string name, int id)
```

Comment:
We can remove this constructor?

Reply:
agreed, we should keep `WorkerInfo(std::string name, int64_t id)` as the only constructor.

Reply:
removed that constructor.
there are sites that passed int as worker_id, I need to convert them to int64_t to match the int64_t constructor that does an overflow check.